src/Symfony/Component/Process/Tests/AbstractProcessTest.php

```diff
@@ -838,10 +860,10 @@ public function testMethodsThatNeedATerminatedProcess($method)
             $process->stop(0);
             $this->fail('A LogicException must have been thrown');
```

Comment:
These fail conditions in the tests you changed are not useless pretty much. So they can be removed.